_includes/v20.1/misc/movr-workflow.md

```diff
@@ -1,23 +1,42 @@
-The workflow for MovR is as follows (with approximations of the corresponding SQL for each step):
+The workflow for MovR is as follows (with approximations of the corresponding SQL and some executable examples for each step):
 
 1. A user loads the app and sees the 25 closest vehicles:
 
     ~~~ sql
     > SELECT id, city, status, ... FROM vehicles WHERE city = <user location>
     ~~~
```

Comment:
I'd restructure each step to read more like this:

```
1. A user loads the app and sees the 25 closest vehicles.
    
    For example:
    
    {% include copy-clipboard.html %}
    ~~~ sql
    > SELECT id, city, status FROM vehicles WHERE city='amsterdam' limit 25; 
    ~~~
```

Reply:
I restructured the wording/format of the steps.